main.go

```diff
@@ -91,6 +91,11 @@ func doMain(args []string) {
 					Name:  "tenant_id",
 					Usage: "Tenant ID (optional) - propagate for just a single tenant.",
 				},
+				cli.DurationFlag{
+					Name:  "throttle",
+					Usage: "`DURATION` to wait for each reindexing batch request",
+					Value: time.Millisecond * 25,
```

Comment:
could we have a const out of it? good news is: you can use it in addressing comments above :)